### DISCUSSION TITLE
Running a bat file through the script

### DISCUSSION THREAD

**Discussion by vjt4** [AutoDarkMode/Windows-Auto-Night-Mode#826] [Q&A]
### Description
      I literally just want to run a script when switching, to change my right click theme. I've tried "CMD" command, i've tried the - - syntax. Can i just get tips on running BATfiles on switch?
`Enabled: true
Component:
  Scripts:
    - Name: cmd example
      Command: powershell
      ArgsLight:
        ['D:\Programs Misc\Nilesoft Shell\imports\Themes\Switch_to_Latte.bat']
      ArgsDark:
        ['D:\Programs Misc\Nilesoft Shell\imports\Themes\Switch_to_Mocha.bat']
      AllowedSources: [Any]`
      

### Expected Behavior

Run a bat file on switch. (the batch files work independently, when manually run:
`@echo off
taskkill /f /im explorer.exe
copy /y Mocha-Peach.nss "D:\Programs Misc\Nilesoft Shell\imports\theme.nss"
explorer.exe
exit 0`

### Log Data

[service.log](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/files/14377097/service.log)
[config yaml githubdontsupportyamluploads.txt](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/files/14377104/config.yaml.githubdontsupportyamluploads.txt)

### Commit Hash, Version and Windows Build

- Commit: `cf7a0fe`
- Service/App: `10.4.1.1`
- Updater: `3.1.4`
- Shell: `1.3.3.0`
- .Net: `7.0.5`
- Windows: `22631.3155`
![image](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/54646079/12dffb47-115e-40b4-8740-0e6c14d22e5f)

**Comment by Spiritreader** [comment]
You can find examples on how to do this here:

https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/wiki/How-to-add-custom-scripts

**Comment by vjt4** [comment]
gotcha i've run it using those examples as a guide and its a no go.

I literally just want to run a bat file. But upon change nothing happens, a terminal window doesn't even open.

The script also obviously works in cmd when run manually.

**Comment by vjt4** [comment]
`Enabled: true
Component:
  Scripts:
    - Name: Shellswitch
      Command: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
      ArgsLight:
        ['D:\Programs Misc\Nilesoft Shell\imports\Themes\Switch_to_Latte.bat']
      ArgsDark:
        [D:\Programs Misc\Nilesoft Shell\imports\Themes\Switch_to_Mocha.bat]
      AllowedSources: [Any]`
      
      So what do i need to even get a powershell window to open on theme switch?
